VERSIONS.md

```diff
@@ -0,0 +1,8 @@
+# IAVL x Cosmos SDK
```

Comment:
we should add this to the bottom of the readme

Reply:
done